packages/next/next-server/lib/router/router.ts

```diff
@@ -434,6 +437,11 @@ export default class Router implements BaseRouter {
     as: string,
     options: TransitionOptions
   ): Promise<boolean> {
+    if (!isLocalURL(url)) {
+      window.location.href = url
```

Comment:
Does this work with `mailto:` links and whatnot?

Reply:
yes, didn't manage to test mailto handlers in selenium, but it's just treated as an external link